### PR TITLE
Fix constrast for accessibility

### LINKED ISSUE #7304: [a11y] Address contrast-comparison violations in wcag2aa report for espresso
Now that https://github.com/nasa/openmct/pull/7047 is complete, we've disabled the contrast-comparison ruleset to address in future iterations.

This ticket captures the work necessary to disable the rule in our tests without necessary addressing every violation in the snow theme

### PR DESCRIPTION
**Pull request by charlesh88**
Closes #7304

### Describe your changes:
- Theme constants changed to pass all WCAG AA level a11y test requirements.
- Event display approach modified to include background colors for constrast reasons.
- Theme colors modified and constrast verified via Wave a11y browser plugin.

@unlikelyzero to investigate: 
- add a header test
- wonky resolution of 1440p
- we need to generate a baseline with all browsers before merge
- understand why we gained vertical screen real estate on all views
- Telemetry Table

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Has this been smoke tested?
* [x] Have you associated this PR with a `type:` label? Note: this is not necessarily the same as the original issue.
* [x] Have you associated a milestone with this PR? Note: leave blank if unsure.
* [x] Is this a breaking change to be called out in the release notes?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [x] Changes appear to address issue?
* [x] Reviewer has tested changes by following the provided instructions?
* [x] Changes appear not to be breaking changes?
* [x] Appropriate automated tests included?
* [x] Code style and in-line documentation are appropriate?

### CODE REVIEW

File: e2e/avpFixtures.js [hunk 1]
```diff
@@ -60,7 +60,7 @@ exports.scanForA11yViolations = async function (page, testCaseName, options = {}
   const builder = new AxeBuilder({ page });
   builder.withTags(['wcag2aa']);
   // https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md
-  builder.disableRules(['color-contrast']);
+  //builder.disableRules(['color-contrast']);
```
Comment (unlikelyzero) [inline e2e/avpFixtures.js:63]: this will eventually be deleted

**Comment by unlikelyzero** [comment]
@charlesh88 the following items are not actually marked as disabled in the DOM.

Screenshot 2023-12-20 at 9 36 00 PM

**Comment by shefalijoshi** [comment]
Not needed for Build 9, but deferring this will incur risk for future merges.

**Comment by akhenry** [comment]
@unlikelyzero What do we need to do to get this merged?

**Pull request merged**